index.js

```diff
@@ -0,0 +1,10 @@
+var twitterText = require('twitter-text');
+
+module.exports = function(text, imageAttachments) {
+  var charsLeft = 140;
+
+  // Images (1 or more), use up 24 characters.
+  if (imageAttachments === true) { charsLeft -= 24; };
+
+  return charsLeft - twitterText.getTweetLength(text);
+};
```

Comment:
Definitely. It might be clearer to understand for you if you separate the function instantiation from the export like such:

```js
const twitterText = require('twitter-text');

const TWEET_LENGTH = 140;
const IMAGE_LENGTH = 24;

function calcTwitterText(text, imageAttachments) {
 var charsLeft = TWEET_LENGTH;

 if (imageAttachments === true) { charsLeft -= IMAGE_LENGTH; };

 return charsLeft - twitterText.getTweetLength(text);
};

module.exports = calcTwitterText;
```